cabal-testsuite/PackageTests/Freeze/T9799b/cabal.test.hs

```diff
@@ -0,0 +1,19 @@
+import Test.Cabal.Prelude
+main = cabalTest $ do
+  expectBroken 9799 $ withRepo "repo" $ do
```

Comment:
I think that it would be better to use a narrower scope for the expected failure, since this line will ignore any failure, including a failure in the build command. If I understand correctly, only the last two assertions are expected to fail. They could be changed to `assertOutputDoesNotContain`, with a comment saying that they should be reversed after issue #9799 is resolved. This comment also applies to the other test.

Reply:
Understood, I've updated the tests accordingly.